.github/workflows/pr.yml

```diff
@@ -1,18 +0,0 @@
-name: Pull request
-on:
-  pull_request_target:
-    types:
-      - opened
-      - reopened
-      - edited
-      - synchronize
-
-jobs:
-  lint:
-    runs-on: ubuntu-latest
-    name: Lint
-    steps:
-      - name: Check PR title
```

Comment:
Where did this step go?

Reply:
oops! I added it back in a pr.yml workflow.